app/assets/javascripts/darkswarm/controllers/offcanvas_controller.js.coffee

```diff
@@ -5,7 +5,7 @@ angular.module('Darkswarm').controller "OffcanvasCtrl", ($scope) ->
     $scope.menu.height($(window).height())
 
   $scope.bind = ->
-    $(window).on("resize", $scope.setOffcanvasMenuHeight)
+    $(window).bind("resize", $scope.setOffcanvasMenuHeight)
```

Comment:
jQuery still used here no?

Reply:
Ah you're right! I thought Angular had it's own `bind()` function, but I think it's a bit different. I'll re-assess this bit :+1:

Reply:
Okay I dug a bit deeper and it looks like Angular includes a min version of `jQuery` anyway, called `jqLite`. If `jQuery` is present, Angular will use that, but if it's not it uses it's own `jqLite` implementation which includes methods like `bind()`.

Anyway, I've switched strategy a bit here, and added some explicit teardown in directives that bind events on DOM elements: https://github.com/openfoodfoundation/openfoodnetwork/pull/8005/commits/15b6b1271a92a393088dd35a5a5628b37ff14d00